packages/spindle-ui/src/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,314 @@
+import React from 'react';
+import ChevronRightBold from '../Icon/ChevronRightBold';
+import ChevronLeftBold from '../Icon/ChevronLeftBold';
+import TriangleendRightBold from '../Icon/TriangleendRightBold';
+import TriangleendLeftBold from '../Icon/TriangleendLeftBold';
+import MenuHorizotal from '../Icon/MenuHorizotal';
+
+interface Props extends React.HTMLAttributes<HTMLElement> {
+  current: number;
+  total: number;
+  pageItems?: number;
+  pageCount?: boolean;
+  pagePrevNext?: boolean;
+  pageFirstLast?: boolean;
+  iconLabel?: boolean;
+  onClick: (pageNumber: any) => void;
```

Comment:
nits. 数字受けるってわかってるならばnumberでよさそうですかね？
```suggestion
  onClick: (pageNumber: number) => void;
```